newton/tests/test_rigid_friction_ramp.py

```
@@ -0,0 +1,243 @@
+        model, _ = build_friction_grid(device)
+        solver = solver_fn(model)
+        state_0 = model.state()
+        state_1 = model.state()
+        control = model.control()
+        contacts = model.contacts() if not isinstance(solver, newton.solvers.SolverMuJoCo) else None
+
+        try:
+            viewer = newton.viewer.ViewerGL()
+            viewer.set_model(model)
+            viewer.set_camera(pos=wp.vec3(6.0, -14.0, 10.0), pitch=-22.0, yaw=90.0)
+        except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)